domain/modelagent/state/state.go

```diff
@@ -43,12 +43,12 @@ func (st *State) GetModelAgentVersion(ctx context.Context, modelID model.UUID) (
 	q := `
 SELECT &dbAgentVersion.target_agent_version
 FROM v_model
-WHERE uuid = $M.model_id
+WHERE uuid = $modelIDArgs.model_id
 `
 
 	rval := dbAgentVersion{}
-	args := sqlair.M{
-		"model_id": modelID,
+	args := modelIDArgs{
```

Comment:
This modelIDArgs and the modelNameArgs, even the result can be the same struct type.

Reply:
Maybe, but I don't know if that's a good idea? It feels like we should have separate types for different use cases.